docs/reference/commandline/login.md

```diff
@@ -40,6 +41,15 @@ adding the server name.
 $ docker login localhost:8080
 ```
 
+### Provide a password automatically
+
+To provide a password (e.g. to script a Docker login command), use
+--password-stdin:
```

Comment:
can you put this in back tics (`--password-stdin`)?

Perhaps something like;

    ### Provide a password using STDIN

    To run the `docker login` command non-interactively, you can set the
    `--password-stdin` flag to provide a password through `STDIN`. Using
    `STDIN` prevents the password from ending up in the Shell's history,
    or log-files.

    The following example reads a password from a file, and passes it to the
    `docker login` command using `STDIN`:

    ```bash
    $ cat ~/my_password.txt | docker login --username foo --password-stdin
    ```